src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -100,7 +567,7 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren('c')).toEqual([span('c:2')]);
   });
 
-  it('works on roots with animation priority before deferred work', () => {
+  it('handles interleaved lo-pri and hi-pri work', () => {
```

Comment:
`scheduleAnimationCallback` used to be called `scheduleHighPriWork` but we renamed it to avoid confusion: https://github.com/facebook/react/pull/7466/commits/5cae65bd9283931c67f6e9a5d0c28c6cabe2f40b

I don't really care what we name them either as long as it's not ambiguous. Overloading "hi-pri" and "lo-pri" is, IMO.